src/main/java/in/ashwanthkumar/gocd/slack/SlackPipelineListener.java

```diff
@@ -92,19 +89,7 @@ private SlackAttachment slackAttachment(GoNotificationMessage message, PipelineS
         try {
             List<MaterialRevision> changes = message.fetchChanges(rules);
             for (MaterialRevision change : changes) {
-                sb.append(change.material.description);
-                sb.append("\n");
                 for (Modification mod : change.modifications) {
-                    String url = change.modificationUrl(mod);
```

Comment:
The correct fix here is to upgrade to a real Slack client library (one exists for Java), and use HTML to style these messages to look like the GitHub bot messages, which would cut down clutter a lot.